### DISCUSSION TITLE
How can I change the video that is played in the game on unity using UABEA?

### DISCUSSION THREAD

**Discussion by MiltyD** [nesrak1/UABEA#327] [Q&A]
This is the situation: I decided to remake one game from steam using UABEA for fun. If there were no problems with replacing textures, then when I wanted to change the video file that is being played in the game, I ran into a problem. Just changing the path where the mp4 video opens doesn't work. As a result, I have a question: how to replace the video in the game and is it even possible? (I don't really understand the work of unity and the work of already compiled games, and therefore I admit that this may not be possible)
![screenshot](https://github.com/nesrak1/UABEA/assets/162918825/7c38d619-60b1-457f-a44a-50888e722d78)

**Comment by nesrak1** [comment]
Do you have a .resS path in the dump as well? I think VideoClips are stored there and not in m_OriginalPath (which is probably just metadata and not actually used?)

**Reply by MiltyD** [reply]
Here is the whole dump, but there is no mention of the .resS file, but the .resS file exists in the game folder. What to do?

[Conditioning_01-sharedassets3.assets-307.txt](https://github.com/nesrak1/UABEA/files/15143457/Conditioning_01-sharedassets3.assets-307.txt)
![Снимок экрана 2024-04-28 224030](https://github.com/nesrak1/UABEA/assets/162918825/3b330981-20f6-4870-8e6c-0ba3c3a3d29d)

**Reply by nesrak1** [reply]
My mistake, it seems to be in the resource file. Unfortunately, I don't have a tool for you to edit .resource files to import video at this moment. You can do it manually by adding to the end of the resource file with a hex editor with a video built from a unity project and then change the byte ranges (m_Offset, m_Size), but it's a bit annoying.

**Reply by MiltyD** [reply]
Ok. Will this be fixed in the future? (It's hard to figure it out manually)
